Common/DataModel/EventSelection.h

```diff
@@ -66,20 +66,21 @@ DECLARE_SOA_COLUMN(Sel8, sel8, bool);                                       //!
 DECLARE_SOA_INDEX_COLUMN_FULL(FoundBC, foundBC, int, BCs, "_foundBC");      //! BC entry index in BCs table (-1 if doesn't exist)
 DECLARE_SOA_INDEX_COLUMN_FULL(FoundFT0, foundFT0, int, FT0s, "_foundFT0");  //! FT0 entry index in FT0s table (-1 if doesn't exist)
 DECLARE_SOA_INDEX_COLUMN_FULL(FoundFV0, foundFV0, int, FV0As, "_foundFV0"); //! FV0 entry index in FV0As table (-1 if doesn't exist)
+DECLARE_SOA_INDEX_COLUMN_FULL(FoundFDD, foundFDD, int, FDDs, "_foundFDD");  //! FDD entry index in FDDs table (-1 if doesn't exist)
 } // namespace evsel
 DECLARE_SOA_TABLE(EvSels, "AOD", "EVSEL", //!
```

Comment:
Would it make sense to split this table into three? A common part, and two others for run 2 and run 3.

Reply:
This, I think, is a question directed to @ekryshen :-D @ekryshen, what do you think?

Reply:
In fact I can just compress it to Run3-like table. I am going to remove 
evsel::BBV0A, evsel::BBV0C, evsel::BGV0A, evsel::BGV0C,
evsel::BBFDA, evsel::BBFDC, evsel::BGFDA, evsel::BGFDC,
because they duplicate info in selection array.
I can also remove evsel::MultRingV0A, evsel::MultRingV0C, evsel::SpdClusters. Those were added just for QA purposes and can be calculated on the fly in the QA task if needed.

Let's merge this PR and I work on further improvements of this table.